bentoml/serve.py

```diff
@@ -84,6 +84,13 @@ def serve_development(
     backlog: int = Provide[BentoMLContainer.api_server_config.backlog],
     bentoml_home: str = Provide[BentoMLContainer.bentoml_home],
     reload: bool = False,
+    ssl_keyfile: t.Optional[str] = None,
+    ssl_certfile: t.Optional[str] = None,
+    ssl_keyfile_password: t.Optional[str] = None,
+    ssl_version: t.Optional[int] = None,
+    ssl_cert_reqs: t.Optional[int] = None,
+    ssl_ca_certs: t.Optional[str] = None,
+    ssl_ciphers: t.Optional[str] = None,
```

Comment:
Should we use dependency injection here?

Reply:
+1. Let's inject config here.
```
ssl_keyfile: t.Optional[str] = Provide[BentoMLContainer.api_server_config.ssl.keyfile]
```

Reply:
Aha, I understand now. I'd considered doing it that was to start, but wasn't entirely clear on if `Provide` would default to `None` as expected.